worker/uniter/resolver/mock_test.go

```diff
@@ -86,7 +86,15 @@ func (e *mockOpExecutor) Run(op operation.Operation) error {
 
 type mockOp struct {
 	operation.Operation
-	commit func(operation.State) (*operation.State, error)
+	commit  func(operation.State) (*operation.State, error)
```

Comment:
This mock is cool, GoMock would certainly help here. We are mocking out the run of the operation. We are only interesting the the side effects of its wrappers (e.g. onCommit, onPrepare) which sync the local and remote state (among other things). Indeed most of these test simply test that the states is being synced correctly.